### PR TITLE
Seed corefx repo with files necessary for dependency flow

### PR DESCRIPTION
**Pull request by weshaggard**
This seeds the repo with the necessary files needed to onboard
the dependency flow for arcade. We are still not using these
dependencies in corefx yet but we want to get them flowing.

cc @mmitche @jcagme @alexperovich

### CODE REVIEW

File: Versions.props [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <VersionPrefix>1.0.0</VersionPrefix>
```
Comment (eerhardt) [inline Versions.props]: What is this file for? If this is meant to be the `corefx` version, this isn't correct, right?

**Comment by mmitche** [comment]
Wes it looks like you got two copies of Version.props/Version.details.xml

@eerhardt The Version.props is dependencies.props + version info.  The version info doesn't really apply to corefx though, and so I'm not sure what we should do with it

**Comment by eerhardt** [comment]
> The version info doesn't really apply to corefx though, and so I'm not sure what we should do with it

Let's make it optional then, so corefx (and I assume there will be other repos) don't need it.

**Comment by mmitche** [comment]
@tmat and @weshaggard

File: Versions.props (hunk 1 above)
Comment (weshaggard) [reply Versions.props]: mistake I removed them https://github.com/dotnet/corefx/pull/32402.

Comment (weshaggard) [reply Versions.props]: Sorry the VersionPrefix doesn't imply and will be cleaned up. The Version file being in the root was a mistake I fixed.

**Comment by weshaggard** [comment]
The Versions.props file in the eng directory will end up being used by corefx once we get everything flowing correctly for now it will just be there.

**Comment by tmat** [comment]
I assume that the files in `eng\common` are exact copies from Arcade, correct?

**Comment by weshaggard** [comment]
>I assume that the files in eng\common are exact copies from Arcade, correct?

Yes.

**Pull request merged**